### DISCUSSION TITLE
hanging nodes

### DISCUSSION THREAD

**Discussion by bigfooted** [su2code/SU2#1426] [General]
Somebody have experience with or is working on hanging nodes? su2 will run without problems using a mesh with hanging nodes, but it looks like the hanging nodes are not treated correctly.

**Comment by pcarruscag** [comment]
Do you have a small example?
The dual volume around the hanging node will not be connected to the larger primal element.
It is probably impossible to determine that the elements around the hanging node are connected using only the connectivity info we have at the moment, since we find the neighbors of elements based on faces that share all their nodes.
However we should be able to detect the bad dual volumes that result from this, because they will be open cvs.
